python/tests/tools/test_mcp_tool.py

```diff
@@ -128,31 +132,34 @@ async def test_mcp_tool_from_client(self, mock_client_session: ClientSession, mo
 # Calculator Tool Tests
 class TestAddNumbersTool:
     @pytest.mark.asyncio
-    @pytest.mark.unit
+    @patch.object(MCPTool, "_run")
     async def test_add_numbers_mcp(
-        self, mock_client_session: ClientSession, add_numbers_tool_info: MCPToolInfo, add_result: Callable
+        self,
+        mock__run,  # noqa: ANN001
+        mock_server_params: StdioServerParameters,
+        add_numbers_tool_info: MCPToolInfo,
+        add_result: Callable,
     ) -> None:
-        mock_client_session.call_tool = AsyncMock(return_value=add_result)
-        tool = MCPTool(client=mock_client_session, tool=add_numbers_tool_info)
+        mock__run.return_value = str(add_result)
+        tool = MCPTool(server_params=StdioServerParameters, tool=add_numbers_tool_info)
         input_data = {"a": 5, "b": 3}
 
         result = await tool._run(input_data)
```

Comment:
same as the other comment: https://github.com/i-am-bee/beeai-framework/pull/481/files#r1980840550